src/test/java/org/sagebionetworks/bridge/services/AdminAccountServiceTest.java

```diff
@@ -888,6 +888,75 @@ public void updateAccount_workerCanChangeStatus() {
         assertEquals(retValue.getStatus(), ENABLED);
     }
     
+    @Test
+    public void updateAccount_noChangeToSynapseUserId() {
```

Comment:
This test name indicates no change, but the Synapse user ID does change.

Reply:
Thanks for catching this, renamed